spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -950,12 +950,8 @@
     end
 
     context 'when signed in with another browser' do
-      it 'redirects back to the client app after concurrent session logout' do
+      it 'redirects back to the client app if logging out after concurrent session logout' do
         user = user_with_2fa
-        service_provider = ServiceProvider.find_by(issuer: OidcAuthHelper::OIDC_IAL1_ISSUER)
```

Comment:
This `link_identity` code wasn't strictly necessary so was removed to avoid excessive duplication in the new, similar test cases being added.